crates/sncast/src/response/structs.rs

```diff
@@ -24,6 +26,30 @@ where
     serializer.serialize_str(&format!("{value:#}"))
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Deserialize, CairoSerialize)]
+pub struct Address(pub Felt);
```

Comment:
I'm not a big fan of using the `Address` name since it is also used for `class_hash`. Generally without additional context, it could be confusing to understand why it was introduced. Apart from this, I think we should also print `transaction_hash` with padded `0` for consistency.

Reply:
Renamed to `PaddedFelt`